mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -252,7 +250,7 @@ class LinearRegression @Since("1.3.0") (@Since("1.3.0") override val uid: String
           model,
           Array(0D),
           Array(0D))
-        return copyValues(model.setSummary(trainingSummary))
+        return model.setSummary(trainingSummary)
```

Comment:
Yes, this is due to we don't have excellent test coverage ...
I will add test case after collect comments.